frontend/src/libs/components/input/input.tsx

```diff
@@ -5,35 +5,61 @@ import {
   type FieldValues,
 } from 'react-hook-form';
 
+import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import { useFormController } from '~/libs/hooks/hooks.js';
 
+import styles from './styles.module.scss';
+
 type Properties<T extends FieldValues> = {
   control: Control<T, null>;
   errors: FieldErrors<T>;
-  label: string;
+  label?: string;
   name: FieldPath<T>;
   placeholder?: string;
   type?: 'text' | 'email' | 'password';
+  isDisabled?: boolean;
 };
 
 const Input = <T extends FieldValues>({
   control,
   errors,
-  label,
+  label = '',
   name,
   placeholder = '',
   type = 'text',
+  isDisabled,
 }: Properties<T>): JSX.Element => {
   const { field } = useFormController({ name, control });
 
   const error = errors[name]?.message;
   const hasError = Boolean(error);
+  const hasValue = Boolean(field.value);
+  const hasLabel = Boolean(label);
+  const inputStyles = [
+    styles.input,
+    hasValue && styles.active,
```

Comment:
Styles active should be applied not when the filed has value, but when field is focused

Reply:
maybe wrong naming for classes, now is filled applied for focus(with blue border) and active just for darker text inside input

Reply:
fixed